arch/aarch64/smp.cc

```diff
@@ -47,7 +46,7 @@ void smp_init()
     if (nr_cpus < 1) {
         abort("smp_init: could not get cpus from device tree.\n");
     }
-    debug("%d CPUs detected\n", nr_cpus);
+    debugf("%d CPUs detected\n", nr_cpus);
```

Comment:
I think because the C++ version of `debug()` became no longer a template and we had already C versions of `debug()` and `debugf()` and we had a conflict.